package/boot/uboot-rockchip/src/of-platdata/roc-cc-rk3328/dt-decl.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * DO NOT MODIFY
```

Comment:
Are these files automatically generated to prevent the inclusion of swig?
If so please mention this in the commit message.

Reply:
> Are these files automatically generated to prevent the inclusion of swig?

yes